docs/source/pip-comparison.rst

```diff
@@ -0,0 +1,26 @@
+===================
+Conda for pip users
```

Comment:
I had thought the [Rosetta Stone](https://en.wikipedia.org/wiki/Rosetta_Stone) was pretty famous. Anyway it's influenced by https://github.com/sympy/sympy/wiki/Git-hg-rosetta-stone.